flink-table-store-core/src/main/java/org/apache/flink/table/store/file/manifest/ManifestFile.java

```diff
@@ -112,26 +112,23 @@ public void delete(String fileName) {
         FileUtils.deleteOrWarn(pathFactory.toManifestFilePath(fileName));
     }
 
-    private class ManifestEntryBulkWriterFactory implements BulkWriter.Factory<ManifestEntry> {
+    private class ManifestEntryFormatWriterFactory implements FormatWriter.Factory<ManifestEntry> {
 
         @Override
-        public BulkWriter<ManifestEntry> create(FSDataOutputStream out) throws IOException {
-            return new BaseBulkWriter<>(writerFactory.create(out), serializer::toRow);
+        public FormatWriter<ManifestEntry> create(Path path) throws IOException {
+            return new BaseFormatWriter<>(writerFactory.create(path), serializer::toRow);
         }
     }
 
-    private class ManifestEntryWriter extends BaseFileWriter<ManifestEntry, ManifestFileMeta> {
-
-        private final FieldStatsCollector statsCollector;
+    private static class ManifestEntryWriter
+            extends BaseFileWriter<ManifestEntry, ManifestFileMeta> {
 
         private long numAddedFiles = 0;
         private long numDeletedFiles = 0;
 
-        ManifestEntryWriter(BulkWriter.Factory<ManifestEntry> writerFactory, Path path)
+        ManifestEntryWriter(FormatWriter.Factory<ManifestEntry> writerFactory, Path path)
                 throws IOException {
             super(writerFactory, path);
-
-            this.statsCollector = new FieldStatsCollector(partitionType);
```

Comment:
The upper layer `ManifestEntryWriter` don't need to handle the column statistic now.